### ISSUE TITLE
[CAIA A11y Research]: Round 2: Veteran Support Team: Feedback Mechanism - Rate Your Experience (Oct./Nov. 2023)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#67274]
### Who's on your team?

```bash
Team name: Veterans Support
OCTO-DE product owner: @clantosswett
Product manager: @ATMiddleton
Designer: @UXCamG
Accessibility specialist: @briandeconinck
Content writer: @starlow00
Slack channel:VSP-Contact-Center-Support

## Note to Team 
- This research support ticket was started by @sara-amanda based on the 10/10/2023 Kickoff Meeting Notes, and it will need to be fully completed by the Veterans Support Team. 
- Please let us know the dates specific to assistive technology testing. 

### User Research starting 10/16/2023:
- CAIA a11ys can provide Assistive Technology Research Support.
- First round of research: Desktop and mobile
- Second round of research: Screen readers and assistive technology
...
```

### Do you have an accessibility champion?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

@briandeconinck

### Calendar checking

- [ ] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [ ] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [ ] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

Round 1 (Desktop and Mobile): 10/16/2023 Based on Intake Ticket; Round 2 (Assistive Technology): xx/xx/xxxx

### Supporting artifacts (if any)

- Product outline: [on Zenhub thread](https://app.zenhub.com/workspaces/contact-center-62cdd9546ec1530018209672/issues/gh/department-of-veterans-affairs/va.gov-team/65148)
- User flows: [Embed Userflow](https://www.sketch.com/s/a513ba38-1d26-41d0-9009-92ea729446e2
- Prototype or mockups: [Embed prototype](https://www.sketch.com/s/a513ba38-1d26-41d0-9009-92ea729446e2)
- Content source of truth:
- Research plan: [Rate Tool](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/resources-and-support/rate-tool)
- Any epics/issues that may be helpful:
- Link to testing accounts (please do not paste them in this ticket):
...

### Next steps

- [ ] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [ ] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [ ] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [ ] CAIA Specialists review your conversation guide and research plan
- [ ] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a research notes template
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
## CAIA a11y Future Research Sessions Update

- [x] Task lists created for the Veteran Support Team and CAIA A11ys in this ticket, with links.
- [ ] [Calendar holds are pending an end date - but Nov. 6 has one hold for now](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) for the dates provided:` Nov. 6 - TBD, 2023.`
- [x] Moving to `ready`, as we await official calendar invites for the AT sessions and end date.

cc: @coforma-terry @SarahKay8 @coforma-jamie

**Comment by coforma-terry** [comment]
Hi @UXCamG - just checking in to see if you have any updates for your planned research sessions which you'd mentioned were hoping to start on the 6th? Thanks !

CC: @sara-amanda

**Comment by UXCamG** [comment]
Hi @coforma-terry-   We are scheduled to begin user testing on Nov 6th. The sessions will run from Nov 6th - 13th.

**Comment by sara-amanda** [comment]
## CAIA Invites Received

_CAIA Capacity Calendar Invited, to sync with invite responses and time changes._

### Week 1: Nov. 6 - 9, 2023
_Short week due to the holiday on Friday and Coforma is out Tuesday for Election Day as well._
- Monday, Nov. 6, 2023 from 12-1 p.m. (@coforma-jamie )
- Monday, Nov. 6, 2023 from 1:30-2:30 p.m. (@coforma-jamie )
- Tuesday, Nov. 7, 2023 from 12-1 p.m. (@sara-amanda )
- Tuesday, Nov. 7, 2023 from 1:30 - 2:30 p.m. (@sara-amanda )

### Week 2: Nov. 13 - 17, 2023 (TBD)
- No invites yet.
- TBD

@coforma-terry FYI

**Comment by UXCamG** [comment]
@sara-amanda I have forwarded the remaining invitations to both you and @coforma-jamie .

**Comment by SarahKay8** [comment]
@UXCamG Hello I updated my availability in Sara's comment. May I receive invites for Nov 8, Nov 9 & the 13th?

**Comment by sara-amanda** [comment]
## CAIA Assistive Technology Sessions

Good Morning, @UXCamG! 

We went ahead and updated the schedule comment above, and _declined_ events for `P3`, `P8` and `P9` on `CAIA Capacity Calendar`, due to no Assistive Technology and/or Cognitive Disability. 

### Currently Scheduled Sessions
_We will do our very best to attend and observe from an a11y perspective, the sessions with screen enlargement, in addition to users with cognitive disabilities, as time allows (our Coforma peers are out for a scheduled holiday this Election Day, and we have a condensed week due to Friday being the observed Veterans Day holiday)._ 

### Screen Readers
We are looking forward to the third round of testing, with the upcoming recruitment of users with Screen Readers. We will be able to not only observe from an a11y perspective, but we can also provide SR support, should the user need assistance, during the call. 

> Thank you for including CAIA in your study! 


**Your CAIA Partners,**
@coforma-jamie @SarahKay8 @sara-amanda 

cc: @coforma-terry

**Comment by sara-amanda** [comment]
## New Sessions Received
### CAIA Invites

- CAIA received invites for three more sessions on Nov. 15 (2) and 16 (1). 
- We have added them to our [capacity calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago). 
- We have added them to the [schedule comment in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67274#:~:text=DUE%20TO%20AT-,New%20Sessions%20Added,-P11%20%2D%20Wednesday%2C%20Nov).

### Assistive Tech Confirmation
Can you confirm @UXCamG that these new sessions include Assistive Technology? Thank you for including CAIA!

**Specifically we are looking for the following, but feel free to add additional information your team has collected, such as Cognitive Disability and Role:**
- **Skill level of user:** Novice, Intermediate or Advanced
- **Operating System:** Mac or Windows
- **Assistive Technology being used:** Jaws, NVDA, ZoomText, Dragon, etc.
- **Internet Browser:** Chrome, Firefox, etc.
- **Device:** Tablet, Mobile or Desktop
- _Cognitive Disability_
- _Role_

@coforma-jamie @sara-amanda @SarahKay8 @coforma-terry

**Comment by UXCamG** [comment]
Participants 12 and 13 will be using magnification devices. Here is the [list](https://perigeantechnologies.knack.com/dsva-perigean-project-tracker#dsva-projects-/?view_103_filters=%5B%7B%22field_name%22%3A%22Active%22%2C%22value%22%3A%22Yes%22%2C%22operator%22%3A%22is%22%2C%22field%22%3A%22field_118%22%7D%5D) for the additional sessions.

**Comment by coforma-jamie** [comment]
@UXCamG, we don't have a Perigean login, can you upload the list here? Thanks!

**Comment by UXCamG** [comment]
Sure @coforma-jamie [List in PDF](https://github.com/department-of-veterans-affairs/va.gov-team/files/13366681/202310rateyourexperiencefeedbackmechanism.pdf)

**Comment by coforma-jamie** [comment]
@UXCamG, it looks like you invited CAIA to P11, but P11 doesn't use assistive technology. Can you take us off of that invite? (today at noon)

**Comment by UXCamG** [comment]
@coforma-jamie CAIA I contacted the scheduler to have CAIA removed from the upcoming session. You all should see an update on your calendar.

**Comment by coforma-terry** [comment]
Still waiting on sessions

**Comment by coforma-jamie** [comment]
Update: the team hasn't finalized a date to start yet, and will let us know when dates are in place. :)

**Comment by sara-amanda** [comment]
Happy 2024, @UXCamG ! Just checking in to see if you had a ballpark timeframe on when you might be conducting your future assistive tech screen reader sessions, as we plan our early 2024 research calendar. If you don't know just yet, no worries. We can check back and/or just let us know, so that CAIA a11ys can assist. 

@coforma-terry @coforma-jamie @SarahKay8

**Comment by UXCamG** [comment]
@sara-amanda Happy 2k24 to you and the CAIA team. We are currently in staging for project Rate tool. We are working with Medallia, and I was told that this project is next in queue for staging with the Medallia team. Once in staging, we will begin user testing, but we do not have a definitive timeline for this. I will be sure to circle back once we have determined a timeline for this project.

**Comment by sara-amanda** [comment]
Thank you so much for the update @UXCamG ! We will wait to hear back!

**Comment by coforma-jamie** [comment]
**Update:** This study is still happening, but there's no ETA on when.

From @UXCamG :
> This project is still in progress, and as of now it is in the queue with Medallia. Our team had to pause our efforts with custom surveys due to the implementation of Medallia V2, but we will soon start again, and this project is on their list.

**Comment by coforma-jamie** [comment]
@strelichl I just got an update from @UXCamG - there's still no ETA. She says

> We have so many surveys in our backlog, we may want to close this and revisit later once this is back in scope
